scripts/aws_ecs_benchmarks.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+source "$HOME/.cargo/env"
+
+# assign local ip
+ip=`curl http://169.254.169.254/latest/meta-data/local-ipv4`
+webserver_url=http://"$ip":9000
+da_webserver_url=http://"$ip":9001
+orchestrator_url=http://"$ip":4444
+
+# build
+just async_std build
+sleep 30s
+
+# docker build and push
+docker build . -f ./docker/validator-webserver-local.Dockerfile -t ghcr.io/espressosystems/hotshot/validator-webserver:main-async-std
+docker push ghcr.io/espressosystems/hotshot/validator-webserver:main-async-std
+
+# ecs deploy
+ecs deploy --region us-east-2 hotshot hotshot_centralized -i centralized ghcr.io/espressosystems/hotshot/validator-webserver:main-async-std
+ecs deploy --region us-east-2 hotshot hotshot_centralized -c centralized ${orchestrator_url}
+
+for total_nodes in 10 50 100
+do
+    for da_committee_size in 10 50 100
+    do
+        if [ $da_committee_size -le $total_nodes ]
+        then
+            for transactions_per_round in 1 10 50 100
+            do
+                for transaction_size in 512 4096 #1000000 20000000
```

Comment:
I think it would be nice to be able to run with all these as parameters instead of looping through.  The way it is now looks good for doing a comprehensive run, but while we iterate, it would be good to be able to run with set committee, total nodes, and txn size

Reply:
Yeah we can do it with multiple variable assignment in script, I'll add this in my next PR.